### PR TITLE
schema: update tidb, support multi drop of column and partition

### LINKED ISSUE #699: split drop multiple columns/partitions in one "alter table ... drop" clause
## Feature Request

**Describe the feature you'd like:**

support to split a statement like

```sql
ALTER TABLE tbl DROP PARTITION p1, p2;
```

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve? 
fix #699

### What is changed and how it works?
update dependency version and add unit test

### Check List 

Tests 

 - Unit test

Code changes

Side effects

Related changes

 - Need to be included in the release note

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,8 +1,28 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+cloud.google.com/go v0.38.0/go.mod h1:990N+gfupTy94rShfmMCWGDn0LpTmnzTp2qbd1dvSRU=
+cloud.google.com/go v0.44.1/go.mod h1:iSa0KzasP4Uvy3f1mN/7PiObzGgflwredwwASm/v6AU=
+cloud.google.com/go v0.44.2/go.mod h1:60680Gw3Yr4ikxnPRS/oxxkBccT6SA1yMk63TGekxKY=
+cloud.google.com/go v0.45.1/go.mod h1:RpBamKRgapWJb87xiFSdk4g1CME7QZg3uwTez+TSTjc=
+cloud.google.com/go v0.46.3/go.mod h1:a6bKKbmY7er1mI7TEI4lsAkts/mkhTSZK8w33B4RAg0=
+cloud.google.com/go v0.50.0 h1:0E3eE8MX426vUOs7aHfI7aN1BrIzzzf4ccKCSfSjGmc=
+cloud.google.com/go v0.50.0/go.mod h1:r9sluTvynVuxRIOHXQEHMFffphuXHOMZMycpNR5e6To=
+cloud.google.com/go/bigquery v1.0.1/go.mod h1:i/xbL2UlR5RvWAURpBYZTtm/cXjCha9lbfbpx4poX+o=
```
Comment (csuzhangxc) [inline go.sum:10]: you can run `go mod tidy` to tidy this _go.sum_.

File: pkg/schema/tracker_test.go [hunk 2]
```diff
@@ -211,6 +211,31 @@ func (s *trackerSuite) TestCreateSchemaIfNotExists(c *C) {
...
+	c.Assert(err, IsNil)
+	err = tracker.Exec(ctx, "testdb", `create table foo(a int, b int, c int)
+        partition by range( a ) (
+			partition p1 values less than (1991),
+			partition p2 values less than (1996),
+			partition p3 values less than (2001)
+	    );`)
+	c.Assert(err, IsNil)
+
+	// Now creating a table should be successful
```
Comment (csuzhangxc) [inline pkg/schema/tracker_test.go:231]: update this comment?

File: go.sum (hunk 1 above)
Comment (lance6716) [reply go.sum:10]: strange no use

**Comment by csuzhangxc** [comment]
Can we re-implement this in _release-1.0_?

**Comment by lance6716** [comment]
> Can we re-implement this in _release-1.0_?

OK, only the downstream TiDB part

**Comment by lance6716** [comment]
> Can we re-implement this in _release-1.0_?

seems release-1.0 doesn't reference tidb's code, so nothing to change

**Pull request merged**